ChatApp/View/SwitcheView.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  SwitcherView.swift
+//  chatApp.test
+//
+//  Created by Ana Dzebniauri on 19.04.23.
+//
+
+import UIKit
+
+
+class ColorSchemeSwitcherComponent: UIView {
+    
+    //MARK: - properties
+    @objc private let lightModeItemBackgroundView = UIImageView()
+    private let lightModeItemView = UIImageView()
+    @objc private let darkModeItemBackgroundView = UIImageView()
+    private let darkModeItemView = UIImageView()
+    private let switchingItemStackView = UIStackView()
+    
+    
+    required init?(coder: NSCoder) {
+        fatalError("!")
+    }
+    
+    override required init(frame: CGRect) {
+        super.init(frame: frame)
+        setUp()
+        
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(viewTapped))
+        addGestureRecognizer(tapGestureRecognizer)
+    }
+    
+    convenience init() {
+        self.init(frame: .zero)
+        setUp()
+    }
+    
+    //MARK: - Functions
```

Comment:
"//MARK: Methods" - it will be better to use methods instead of functions since "A function is a set of instructions or procedures to perform a specific task, and a method is a set of instructions that are associated with an object."